x-pack/filebeat/input/httpjson/config_auth.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/elastic/beats/v7/libbeat/common"
 )
 
+const AuthStyleInParams = 1
```

Comment:
Do you need to expose this const (`AuthStyleInParams` -> `authStyleInParams`)?

Reply:
Updated.
Thanks